sos/report/plugins/lxd.py

```diff
@@ -41,8 +41,8 @@ def setup(self):
         lxd_pred = SoSPredicate(self, kmods=lxd_kmods,
                                 required={'kmods': 'all'})
 
-        snap_list = self.exec_cmd('snap list lxd')
-        if snap_list["status"] == 0:
+        if self.policy.package_manager.pkg_by_name(
+                'lxd')['pkg_manager'] == 'snap':
```

Comment:
Theoretically, if one *manually* enables this plugin on a system without `lxd` package installed, we would get `TypeError: 'NoneType' object is not subscriptable` here (am I right?). Not sure how feasible this option is / if we should catch that in the code.